doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -83,6 +83,7 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 .. fota_upgrades_over_ble_additional_information_end
 
 .. fota_upgrades_over_ble_mcuboot_direct_xip_information_start
+.. _mcuboot_direct_xip_mode:
```

Comment:
Isn't it better to move this label before the fota_upgrades_over_ble_mcuboot_direct_xip_information_start label? Wouldn't we end up with conflicting labels once we include the fota_upgrades_over_ble_mcuboot_direct_xip_information_start in another RST file?

Reply:
I tried the link target label before the start label, but it ended up in tears (a doc build failure).

I'd rather have the start label after the heading and when including, then add the heading. I'm ok to change the name of the link target label.

Reply:
I see. Then, I guess we need to move the start label after the section heading. @mkapala-nordic, let us know if this has a potential to break some part of documentation that you added for direct-xip (I think this _start _end labelling sequence isn't used anywhere currently).

Reply:
The `fota_upgrades_over_ble_mcuboot_direct_xip_information_start` and `fota_upgrades_over_ble_mcuboot_direct_xip_information_end` labels are not currently used.
I think they can be safely moved after the heading or even removed if necessary.